qtawesome/iconic_font.py

```diff
@@ -174,6 +176,7 @@ def __init__(self, *args):
         self.painters = {}
         self.fontname = {}
         self.charmap = {}
+        qapplication()
```

Comment:
@dalthviz you are making QTAwesome depend on Spyder, which is not what we want.

We should use QApplication

```python
from qtpy import QtWidgets

# ...

app = QApplication([])

```